fla/models/moba/modeling_moba.py

```
@@ -0,0 +1,347 @@
+        input_ids: torch.LongTensor | None = None,
+        attention_mask: Optional[torch.Tensor] = None,  # noqa
+        inputs_embeds: torch.FloatTensor | None = None,
+        past_key_values: Cache | list[torch.FloatTensor] | None = None,
+        use_cache: bool | None = None,
+        output_attentions: bool | None = None,
+        output_hidden_states: bool | None = None,
+        return_dict: bool | None = None,
+        **kwargs: Unpack[dict],
+    ) -> tuple | BaseModelOutputWithPast:
+        if output_attentions:
+            warnings.warn("`MoBAModel` does not `output_attentions` now, setting it to `False`.")
```

Ruff (0.15.10)

[warning] No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)